test/e2e/bootstrapper_test.go

```diff
@@ -0,0 +1,113 @@
+package e2e
+
+import (
+	"fmt"
+	"github.com/openshift/windows-machine-config-operator/pkg/bootstrapper"
+	"github.com/spf13/pflag"
+	"github.com/stretchr/testify/assert"
+	"golang.org/x/sys/windows/svc"
+	"golang.org/x/sys/windows/svc/mgr"
+	"testing"
+	"time"
+)
+
+var ignitionFilePath string
+var kubeletPath string
+var installDir string
+
+func init() {
+	pflag.StringVar(&ignitionFilePath, "ignition-file", "C:\\Windows\\Temp\\worker.ign", "ign file location")
+	pflag.StringVar(&kubeletPath, "kubelet-path", "C:\\Windows\\Temp\\kubelet.exe", "kubelet location")
+	pflag.StringVar(&installDir, "install-dir", "C:\\k", "Installation directory")
+}
+
+func TestBootstrapper(t *testing.T) {
+	wcb, err := bootstrapper.NewWinNodeBootstrapper(installDir, ignitionFilePath, kubeletPath)
+	assert.Nil(t, err)
+	// Start the bootstrapper, which will start the kubelet service
+	assert.Nil(t, wcb.Start())
+	wcb.Disconnect()
+	fmt.Println("started")
+	time.Sleep(5 * time.Second)
+	// Ensure that the kubelet service is running
+	assert.True(t, svcRunning(t, bootstrapper.ServiceName))
+	wcb, err = bootstrapper.NewWinNodeBootstrapper(installDir, ignitionFilePath, kubeletPath)
+	// "Stop" the bootstrapper which will remove the kubelet service
+	assert.Nil(t, wcb.Stop())
+	assert.Nil(t, err)
+	wcb.Disconnect()
+	fmt.Println("stopped")
+	time.Sleep(5 * time.Second)
+	// Ensure that we've removed the kubelet service
+	assert.False(t, svcExists(t, bootstrapper.ServiceName))
+}
+
+// svcRunning returns true if the service with the name svcName is running
+func svcRunning(t *testing.T, svcName string) bool {
+	state, err := getSvcState(svcName)
+	assert.Nil(t, err)
+	return svc.Running == state
+}
+
+// svcExists returns true with the service with the name svcName is installed, the state of the service does not matter
+func svcExists(t *testing.T, svcName string) bool {
+	svcMgr, err := mgr.Connect()
+	assert.Nil(t, err)
+	defer svcMgr.Disconnect()
+	mySvc, err := svcMgr.OpenService(svcName)
+	if err != nil {
+		return false
+	} else {
+		mySvc.Close()
+		return true
+	}
+}
+
+func getSvcState(svcName string) (svc.State, error) {
```

Comment:
Update the comment. What's the expectation here. Can anyone connect to svcs? or we need to run the `go binary` as admin or as user with certain privilege. I'm talking about Windows OS privileges

Reply:
:+1: